container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1644,7 +1644,7 @@ roles:
   - name: eventgenerator
   run:
     scaling:
-      min: 0
+      min: 1
```

Comment:
Would it be possible to leave this disabled-by-default so that working on the rest of SCF can use a smaller vagrant box?

I'd be happy to take changes to `make/run` to make it easier to start a run with autoscaler turned on, of course.  Could even check for an environment variable if you want to get fancy (and then you can just drop a `.envrc` in a ancestor directory and never think about it again).